libethash/io_posix.c

```diff
@@ -86,9 +90,22 @@ bool ethash_file_size(FILE* f, size_t* ret_size)
 
 bool ethash_get_default_dirname(char* strbuf, size_t buffsize)
 {
+#if defined(__APPLE__)
+	static const char dir_suffix[] = "Ethash/";
```

Comment:
This is not following the [spec](https://github.com/ethereum/wiki/wiki/Ethash-DAG)